### DISCUSSION TITLE
AUC ratio, C_max Ratio

### DISCUSSION THREAD

**Discussion by kyun-gran** [Open-Systems-Pharmacology/Forum#1611] [Questions & Problems]
![캡처](https://github.com/Open-Systems-Pharmacology/Forum/assets/75706166/92023bd5-d120-4546-abee-04e41dc4b300)

Why can't I get the values of AUC ratio and C_max ratio in Digoxin?
I tried many other methods to get the value, but I couldn't get the value only from digoxin. What's the reason?

I'm looking forward to hearing from you.
Thank you

**Comment by PavelBal** [comment]
If digostin is a perpetrator and not a victim, AUC_R and Cmax_R do not make sense in this case.

**Reply by kyun-gran** [reply]
Yes, I'd like to compare drug A with DDI in digoxin.
Drug A is set to perpetrator  and digoxin is set to victim.
Even in this case, A shows AUC_ratio and Cmax_ratio, but not digoxin.
Looking at the above results, it seems that there is drug A instead of atorvastatin.
